shapely/geometry/geo.py

```diff
@@ -18,6 +18,33 @@ def box(minx, miny, maxx, maxy, ccw=True):
         coords = coords[::-1]
     return Polygon(coords)
 
+def anchored_box(anchor='nw',x,y,width,height,ccw=True):
```

Comment:
Arguments should have a space after the comma, e.g.

```
def anchored_box(anchor="nw", x, y, width, height, ccw=True):
```

Reply:
Also, see the error on Travis. You have a non-default argument (x, y, etc.) following a default argument (anchor). The easiest thing is to just remove the default. If we do keep it we'll have to argue about if the default should be NW or SW. ;)